Frameworks/UIKit/UIPasteboard.mm

```diff
@@ -173,6 +171,8 @@ + (void)removePasteboardWithName:(NSString*)pasteboardName {
 /**
  @Status Caveat
  @Notes only NSString, NSURL and UIImage are supported.
+  If any of supported type in pasteboardTypes is supported, return YES.
+  otherwise, return NO. (reference platform also has the same behavior)
```

Comment:
Why bother mentioning this if it's the same thing that happens on the reference platform?